zubhub_frontend/zubhub/src/components/form/dropdown/Dropdown.jsx

```diff
@@ -35,9 +35,9 @@ export default function Dropdown({
     value = valueTemp;
   }
 
-  const labelView = (props, option, { selected }) => {
-    const color = selected ? colors.primary : colors.light;
-    const checked = selected;
+  const labelView = (props, ...rest) => {
+    const color = rest[0].selected ? colors.primary : colors.light;
```

Comment:
I think the object destructing was quite straight forward, if you check the `mui` docs for the combo box I think